code/src/java/pcgen/cdom/base/CDOMObject.java

```diff
@@ -230,7 +230,7 @@ public final Formula remove(FormulaKey key)
 
 	public final Set<FormulaKey> getFormulaKeys()
 	{
-		return formulaChar == null ? Collections.<FormulaKey> emptySet()
+		return formulaChar == null ? Collections.emptySet()
```

Comment:
AFAICT this won't generate a new warning since the type is inferred but I might not understand Java / noticed the warning?